packages/core/src/utils.ts

```diff
@@ -504,27 +501,33 @@ export const storeCanonicalConfig: StoreCanonicalConfig = async (
   const response: {
     status: number
     data: string[]
-  } = await axios.post(`${fetchSphinxManagedBaseUrl()}/api/pin`, {
-    apiKey,
-    orgId,
-    configData,
-  })
+  } = await axios
+    .post(`${fetchSphinxManagedBaseUrl()}/api/pin`, {
+      apiKey,
+      orgId,
+      configData,
+    })
+    .catch((err) => {
```

Comment:
Just refactored this to match the pattern I used for the call to fetch the artifacts.